packages/js/core-client/README.md

```diff
@@ -6,3 +6,382 @@
 <br/>
 <br/>
 The Polywrap JavaScript core client.
+
+## Installation
+
+```bash
+npm install --save @polywrap/core-client-js
+```
+
+## Usage
+
+### Instantiate the client
+```ts
+import { PolywrapCoreClient } from "@polywrap/core-client-js";
+
+const config = { ... };
+const client = new PolywrapCoreClient(config);
+```
+
+### Invoke a wrapper
+
+```ts
+await client.invoke({
+  uri: "ens/rinkeby/helloworld.dev.polywrap.eth",
```

Comment:
```suggestion
  uri: "ens/helloworld.dev.polywrap.eth",
```

Reply:
also: i would also suggest this in the `client/README.md` but gh doesn't allow me